packages/theme/src/cli/commands/theme/pull.ts

```diff
@@ -54,20 +54,21 @@ export default class Pull extends ThemeCommand {
     }),
   }
 
-  static cli2Flags = ['theme', 'development', 'live', 'nodelete', 'only', 'ignore', 'force']
+  static cli2Flags = ['theme', 'development', 'live', 'nodelete', 'only', 'ignore', 'force', 'development-theme-id']
 
   async run(): Promise<void> {
-    let {flags} = await this.parse(Pull)
+    const {flags} = await this.parse(Pull)
     const store = ensureThemeStore(flags)
     const adminSession = await ensureAuthenticatedThemes(store, flags.password)
 
-    if (flags.development) {
-      const theme = await new DevelopmentThemeManager(adminSession).find()
-      flags = {
-        ...flags,
-        development: false,
-        theme: theme.id.toString(),
+    const developmentThemeManager = new DevelopmentThemeManager(adminSession)
+    const theme = await (flags.development ? developmentThemeManager.find() : developmentThemeManager.fetch())
```

Comment:
maybe im missing something but does this mean even when a developer uses theme pull to get a live theme, the `developmentThemeManager` is used?

Reply:
Great question!
Yes, we pass the `--development-theme-id` every time a Development Theme exists, to make sure that CLI 2 always has the most recent ID in its DB (see description).
The `--theme` flag can still include the live theme though :)